openapi/security-secretstore-setup.yaml

```diff
@@ -0,0 +1,124 @@
+      type: object
+      properties:
+        apiVersion:
+          description: "A version number shows the API version in DTOs."
+          type: string
+          example: v3
+        requestId:
+          description: "Uniquely identifies the request that resulted in this response."
+          type: string
+          format: uuid
+          example: "e6e8a2f4-eb14-4649-9e2b-175247911369"
+        message:
+          description: "A field that can contain a free-form message, such as an error message."
+          type: string
+        statusCode:
+          description: "A numeric code signifying the operational status of the response."
+          type: integer
+    ErrorResponse:
+      allOf:
+        - $ref: '#/components/schemas/BaseResponse'
+      description: "A response type for returning a generic error to the caller."
+      type: object
+
+  headers:
+    correlatedResponseHeader:
+      description: "A response header that returns the unique correlation ID used to initiate the request."
+      schema:
+        type: string
+        format: uuid
+      example: "14a42ea6-c394-41c3-8bcd-a29b9f5e6835"
+  examples:
+    400Example:
+      value:
+        apiVersion: "v3"
+        requestId: "73f0932c-0148-11eb-adc1-0242ac120002"
+        statusCode: 400
+        message: "Bad Request"
+    401Example:
+      value:
+        apiVersion: "v3"
+        requestId: "73f0932c-0148-11eb-adc1-0242ac120002"
+        statusCode: 401
+        message: "Unauthorized"
+    403Example:
+      value:
+        apiVersion: "v3"
+        requestId: "73f0932c-0148-11eb-adc1-0242ac120002"
+        statusCode: 403
+        message: "Forbidden"
+    404Example:
+      value:
+        apiVersion: "v3"
+        requestId: "84c9489c-0148-11eb-adc1-0242ac120002"
+        statusCode: 404
+        message: "Not Found"
+    500Example:
+      value:
+        apiVersion: "v3"
+        statusCode: 500
+        message: "Interval Server Error"
+
+
+paths:
+  /token/entityId/{entityId}:
+    parameters:
+      - name: entityId
+        in: path
+        required: true
+        schema:
+          type: string
+        description: "Specifies the id of the secret store entity"
+    put:
+      summary: Create a new secret store client token for the specified entity id and saved in the token file
+      responses:
+        '200':
+          description: "OK"
+          headers:
+            X-Correlation-ID:
+              $ref: '#/components/headers/correlatedResponseHeader'
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/BaseResponse'
+        '400':
+          description: "Bad Request"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+              examples:
+                400Example:
+                  $ref: '#/components/examples/400Example'
+        '403':
+          description: "Bad Request"
```

Comment:
is there any chance to return 403?
if yes, modify the description to "Forbidden"

Reply:
Yes, it might return 403 if the privileged token has no permission to create the token.